_studio/shared/umc/codec/h264_dec/src/umc_h264_au_splitter.cpp

```diff
@@ -40,9 +40,16 @@ SeiPayloadArray::SeiPayloadArray()
 SeiPayloadArray::SeiPayloadArray(const SeiPayloadArray & payloads)
 {
     size_t count = payloads.GetPayloadCount();
-    for (size_t i = 0; i < count; i++)
+    if (count == 0)
```

Comment:
```m_payloads``` is a ```std::vector``` which is default constructable and well-defined in case if ```count==0```. you don't need to change anything